src/pages/Home/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import getToken from '../../lib/tokens';
+
+const saveToken = (e) => {
+  e.preventDefault();
+  const token = getToken();
+  localStorage.setItem('token', token);
+  window.location.reload(false);
```

Comment:
Based on this AC:
> Once the token has been created and saved, the user is redirected to the "list" view

Currently your saveToken does not handle the above use case. It doesn't navigate away which allows the user to keep clicking on the button to create new tokens to replace the one in local storage. Here is where you would use [`useHistory()`](https://reactrouter.com/web/api/Hooks/usehistory) like @abowler2 had recommended. 

![Screen Cast 2021-07-24 at 7 30 56 AM](https://user-images.githubusercontent.com/6759658/126881118-187f2d6e-8448-41a1-97a9-413e8ff14575.gif)

Reply:
So I was not able to recreate this error in my environment, I am getting redirected as we intended. I wonder why it could be behaving differently for me? Regardless it sounds like `window.location()` is not the optimal way to go and we can implement a change to this per @abowler2 recommendation.